src/store/module/product.module.js

```diff
@@ -0,0 +1,56 @@
+const product = {
+  namespaced: true,
+  state () {
+    return {
+      products: [],
+      items: {},
```

Comment:
what is items here ?

Reply:
items are used to recieve a value which is inputed in the form of products.vue

Reply:
Sorry! I didn't get you. Can you make it clear